packages-exp/auth-exp/src/core/mfa/mfa_info.test.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProviderId } from '@firebase/auth-types-exp';
+import { FirebaseError } from '@firebase/util';
+import { expect, use } from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { testAuth } from '../../../test/mock_auth';
+import { Auth } from '../../model/auth';
+import { MultiFactorInfo } from './mfa_info';
+import { PhoneMfaEnrollment } from '../../api/account_management/mfa';
+
+use(chaiAsPromised);
+
+describe('core/mfa/mfa_info', () => {
+  let auth: Auth;
+
+  beforeEach(async () => {
+    auth = await testAuth();
+  });
+
+  describe('_fromServerResponse', () => {
+    context('phone enrollment', () => {
+      const date = Date.now();
+      const enrollmentInfo = {
+        mfaEnrollmentId: 'uid',
+        enrolledAt: Date.now(),
```

Comment:
There's an ever so slight chance that this and `date` definition above fall onto a different millisecond boundary.